packages/core/src/i18n/translations/en.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  'required': '(*Required)',
+  'max-chars': '(Max. {{length}} characters)',
+};
```

Comment:
In a fully internationalized component library, these translations will likely live somewhere else as part of the platform's larger internationalization strategy.